src/bson/bson-version.h

```diff
@@ -37,30 +37,30 @@
  *
  * BSON minor version component (e.g. 2 if %BSON_VERSION is 1.2.3)
  */
-#define BSON_MINOR_VERSION (17)
+#define BSON_MINOR_VERSION (20)
 
 
 /**
  * BSON_MICRO_VERSION:
  *
  * BSON micro version component (e.g. 3 if %BSON_VERSION is 1.2.3)
  */
-#define BSON_MICRO_VERSION (0)
+#define BSON_MICRO_VERSION (1)
 
 
 /**
  * BSON_PRERELEASE_VERSION:
  *
  * BSON prerelease version component (e.g. pre if %BSON_VERSION is 1.2.3-pre)
  */
-#define BSON_PRERELEASE_VERSION ()
+#define BSON_PRERELEASE_VERSION (pre)
 
 /**
  * BSON_VERSION:
  *
  * BSON version.
  */
-#define BSON_VERSION (1.17.0)
+#define BSON_VERSION (1.20.1-pre)
```

Comment:
You've copied from c driver master. Please redo this and copy from the 1.20.0 tag. This explains why you have BSON_HAVE_ARC4RANDOM_BUF.